config.js

```diff
@@ -98,6 +98,14 @@ var config = {
     // Disables self-view settings in UI
     // disableSelfViewSettings: false,
 
+    // Enables the screensharing capture feature.
+    // enableScreenshotCapture: false,
+
+    // The mode for the screenshot capture feature.
+    // Can be either 'recording' - screensharing screenshots are taken only when the recording is also on,
+    // or 'always' - screensharing screenshots are always taken.
+    // screenshotCaptureMode: 'recording',
```

Comment:
maybe group the 2 configs:  
screenshotCapture: { enabled: boolean, mode: string }